CHANGELOG.md

```diff
@@ -1,6 +1,29 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [0.6.5] - 2024-11-08
+### Added
+  * Update to Golang 1.22.
```

Comment:
I don't think the Golang update should be here (it's not a product feature per-se and we've never added it before).

Reply:
We have done it before.

Agree that Golang update is not a product feature, so it's not relevant for end-users, but it may very well be for developers out there trying to build Sysbox, which explains why most projects in our space include this in their release changelogs (containerd, moby, etc.).

Reply:
OK, but we've updated Golang many times before but didn't mention it in the release notes.

Reply:
True, we have mentioned it only once or twice, but we should start doing it consistently from now on, IMO.